### DISCUSSION TITLE
SA:MP support

### DISCUSSION THREAD

**Discussion by ghosty2004** [cleolibrary/CLEO-Redux#72] [Ideas]
Hi! That would be cool if this cleo version will support the **SA:MP version** (ex: listen for incoming RakNet events and modify the callback) It's possible to add sa:mp support for this ?

**Comment by x87** [comment]
See https://github.com/cleolibrary/CLEO-Redux/issues/92

There must be a community effort to bring CLEO Redux functionality beyond currently supported games.
